JASP-Engine/JASP/R/correlation.R

```diff
@@ -671,11 +689,22 @@ Correlation <- function(jaspResults, dataset, options){
   
   # Report errors as footnotes
   for(i in seq_along(errors)){
+    # display general errors (i.e., too much missing data, etc. identified from .hasErrors)
     if(is.list(errors[[i]])){
       pair <- pairs[[i]]
-      statsNames <- statsNames[statsNames != "sample.size"]
-      colNames <- paste(pair[1], statsNames, sep = "_")
-      mainTable$addFootnote(message = errors[[i]]$message, colNames = colNames, rowNames = pair[2])
+      colNames <- statsNames[statsNames != "sample.size"]
+      colNames <- paste(pair[2], colNames, sep = "_")
+      mainTable$addFootnote(message = errors[[i]]$message, colNames = colNames, rowNames = pair[1])
+    }
+    
+    # display test errors (i.e., during calculating results, such as failure to invert a correlation matrix, etc.)
+    for(test in c("pearson", "spearman", "kendall")){
```

Comment:
a space after 
- `for`
- `if`

Add a space at the end before `{` as well please